connect/server/src/main/scala/org/apache/spark/sql/connect/execution/ExecuteThreadRunner.scala

```diff
@@ -115,7 +116,8 @@ private[connect] class ExecuteThreadRunner(executeHolder: ExecuteHolder) extends
         case e: Throwable =>
           logDebug(s"Exception in execute: $e")
           // Always cancel all remaining execution after error.
-          executeHolder.sessionHolder.session.sparkContext.cancelJobsWithTag(executeHolder.jobTag)
+          executeHolder.sessionHolder.session.sparkContext.cancelJobsWithTag(executeHolder.jobTag,
+            "Exception in execute:" + ExceptionUtils.getStackTrace(e))
```

Comment:
Huh, looks like `ExceptionUtils.getStackTrace` is approximately equivalent to Spark's own `Utils.exceptionString` but it seems that we're not very consistent about using one vs. the other.

I think you should add an extra space after `execute:` so this concatenates properly.

Reply:
Sounds good, I also switched to using `Utils.exceptionString` because it seems to be more commonly used than the other in the codebase.